src/Solver/time_stepping/AbstractGhostTimeCluster.cpp

```diff
@@ -116,4 +116,13 @@ void AbstractGhostTimeCluster::printTimeoutMessage(std::chrono::seconds timeSinc
         << "correctionsSinceSync = " << neighbor.ct.stepsSinceLastSync;
   }
 }
+
+ClusterTimes& AbstractGhostTimeCluster::getClusterTimes(){
+  return ct;
+}
+
+std::vector<seissol::time_stepping::NeighborCluster>* seissol::time_stepping::AbstractGhostTimeCluster::getNeighborClusters(){
```

Comment:
Reference? Or do you need this vector somewhere?

Reply:
The vector is needed. If I recall it, @krenzland gave me this fix when the reference was not working for what we were thinking of doing.